python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -339,6 +340,29 @@ async def setup_laser(
         await self.change_laser_optical_configuration(optical_configuration)
         await self.change_laser_wavelength(wavelength, use_projector)
 
+    async def get_projector_setup(self) -> tuple:
+        """Get configuration of flatfield projector
+
+        Returns
+        -------
+            list : configuration details
```

Comment:
The type hint says it is a tuple, but here you say it is a list. 

You are returning a tuple with known size and type so, please, update the docstring.

Also, you might want to state what each position of the tuple contains.